src/main/scala/Numbers.scala

```diff
@@ -12,9 +11,11 @@ class Numbers {
   val sextillion = pow(10, 21)
 
 
-  def shortScale(x: Double): String = {
+
+  def shortScale(x: Long): String = {
     x match {
-      case x if (x < million) => (x / 1000).toInt.toString + " thousands and " + (x % 1000).toInt.toString
+      case x if(x<thousand)=>x.toInt.toString
```

Comment:
Intellij or scala style should've been highlighting that the conditional check doesn't need to be wrapped in parentheses.  case x if x<thousand=>x.toInt.toString  works fine as well